pkg/cli/cluster_migrate_multinode_storage.go

```diff
@@ -31,79 +30,33 @@ type migrateOpts struct {
 	ekcoAddress    string
 	readyTimeout   time.Duration
 	migrateTimeout time.Duration
+	minimumNrNodes int
 }
 
 func NewClusterMigrateMultinodeStorageCmd(cli CLI) *cobra.Command {
 	opts := migrateOpts{log: cli.Logger()}
 	cmd := &cobra.Command{
-		Use:          "migrate-multinode-storage",
-		Short:        "Migrate persistent volumes from 'scaling' to 'distributed' storage classes.",
-		SilenceUsage: true,
-		RunE: func(cmd *cobra.Command, args []string) error {
-			cfg, err := config.GetConfig()
-			if err != nil {
-				return fmt.Errorf("failed to get kubernetes config: %w", err)
-			}
-			client, err := kubernetes.NewForConfig(cfg)
-			if err != nil {
-				return fmt.Errorf("failed to get kubernetes client: %w", err)
-			}
-			opts.log.Printf("reading ekco operator credentials.")
-			if opts.authToken, err = getEkcoStorageMigrationAuthToken(cmd.Context(), client); err != nil {
-				return fmt.Errorf("failed to get ekco storage migration auth token: %w", err)
-			}
-			opts.log.Printf("finding ekco operator http address.")
-			if opts.ekcoAddress, err = getEkcoAddress(cmd.Context(), client); err != nil {
-				return fmt.Errorf("failed to get ekco address: %w", err)
+		Use:   "migrate-multinode-storage",
+		Short: "Migrate persistent volumes from 'scaling' to 'distributed' storage classes.",
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			if opts.minimumNrNodes <= 0 {
+				return fmt.Errorf("--minimum-number-of-nodes flag is required to be > 0.")
 			}
+			cmd.SilenceUsage = true
+			return nil
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
 			return runStorageMigration(cmd.Context(), opts)
 		},
 	}
 	cmd.Flags().DurationVar(&opts.readyTimeout, "ready-timeout", 10*time.Minute, "Timeout waiting for the cluster to be ready for the storage migration.")
 	cmd.Flags().DurationVar(&opts.migrateTimeout, "migrate-timeout", 8*time.Hour, "Timeout waiting for the storage migration to finish.")
+	cmd.Flags().IntVar(&opts.minimumNrNodes, "minimum-number-of-nodes", 0, "Indicates the desired minimum number of nodes to start the migration.")
+	cmd.Flags().StringVar(&opts.ekcoAddress, "ekco-address", "ecko-operator.kurl:"+strconv.Itoa(ekcoPort), "The address of the ekco operator.")
```

Comment:
_Editor Comment_: `ecko-address` should match the service [name]( https://github.com/replicatedhq/kURL/pull/4596/files#diff-9bbbdc3947acfe58c9ea00079d4346d7f40b1163818f11fdd589ebf8965a199aR4)